hail/src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -1534,3 +1534,13 @@ object RVD {
     fileData
   }
 }
+
+case class BroadcastRVD(@transient val backend: SparkBackend, @transient val rvd: RVD) extends Serializable {
+  private[this] val crdd = rvd.crdd
+  private[this] val bcPartitions = backend.broadcast(rvd.crdd.partitions)
```

Comment:
longer-term, we should serialize the partition to a TBinary inside the context and use that.